README.md

```diff
@@ -51,77 +51,95 @@ on `initSync` to generate the table definition and write it to CrateDB.
+            ],
+        },
+        notificationPreferences: {
+            bsonType: 'array',
+            items: {
+                bsonType: 'string',
+                enum: [
+                    'newMatchingRFQ',
+                    'activityOnRFQWhereParticipant',
+                    'activityOnRFQBySameOrgUsers',
+                ],
             },
-          },
-          name: { bsonType: 'string' },
-          isPrimary: { bsonType: 'bool' },
         },
-      },
-    },
-    integrations: {
-      bsonType: 'object',
-      additionalProperties: true,
-      properties: {
-        stripe: {
-          bsonType: 'object',
-          additionalProperties: true,
-          properties: {
-            priceId: {
-              bsonType: 'number',
+        addresses: {
+            bsonType: 'array',
+            items: {
+                bsonType: 'object',
+                additionalProperties: false,
+                properties: {
+                    address: {
+                        bsonType: 'object',
+                        additionalProperties: false,
+                        properties: {
+                            street: { bsonType: 'string' },
+                            city: { bsonType: 'string' },
+                            county: { bsonType: 'string' },
+                            state: { bsonType: 'string' },
+                            zip: { bsonType: 'string' },
+                            country: { bsonType: 'string' },
+                            latitude: { bsonType: 'number' },
+                            longitude: { bsonType: 'number' },
+                        },
+                    },
+                    name: { bsonType: 'string' },
+                    isPrimary: { bsonType: 'bool' },
+                },
             },
-            subscriptionStatus: {
-              bsonType: 'string',
+        },
+        integrations: {
+            bsonType: 'object',
+            additionalProperties: true,
+            properties: {
+                stripe: {
+                    bsonType: 'object',
+                    additionalProperties: true,
+                    properties: {
+                        priceId: {
+                            bsonType: 'number',
+                        },
+                        subscriptionStatus: {
+                            bsonType: 'string',
+                        },
+                    },
+                },
             },
-          },
         },
-      },
-    },
-    metadata: {
-      bsonType: 'object',
+        metadata: {
+            bsonType: 'object',
+        },
     },
-  },
 }
 
-convertSchema(schema, '"doc"."foobar"')
+convertSchema(schema, '"doc"."foobar"', {
+    overrides: [
+        { path: 'addresses.address.latitude', bsonType: 'double' },
+        { path: 'addresses.address.longitude', bsonType: 'double' },
+        {
+            path: 'description',
+            flags: ['notNull', 'indexOff', 'columnStoreOff'],
```

Comment:
What would be a use case where you want to turn off column store and not indexing or would turning these off always go hand in hand?